lib/spack/spack/test/cmd/config.py

```diff
@@ -239,6 +240,7 @@ def test_config_add_ordered_dict(mutable_empty_config):
 """
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="Error on Win")
```

Comment:
Would it be possible to revisit some of the failing tests here and see if they now pass? For example the actions here look very similar to those in `test_config_add_ordered_dict` (just above), so I'm surprised this test fails.

Reply:
Added to lists of tests I'm re-enabling.